Src/FluentAssertions/Collections/CollectionAssertions.cs

```diff
@@ -936,9 +948,32 @@ public AndConstraint<TAssertions> BeInDescendingOrder(IComparer<object> comparer
         /// <param name="becauseArgs">
         /// Zero or more objects to format using the placeholders in <see cref="because" />.
         /// </param>
+        /// <remarks>
+        /// Empty and single element collections are considered to be ordered both in ascending and descending order at the same time.
+        /// </remarks>
+        [Obsolete("Use NotBeInAscendingOrder instead")]
```

Comment:
For other methods marked with `Obsolete`, we also added 
```
[EditorBrowsable(EditorBrowsableState.Never)]
```
This will hide the method when using IntelliSense to browse available methods.
Could I ask you add those as well?